pkg/server/sandbox_stop.go

```diff
@@ -47,15 +47,19 @@ func (c *criContainerdService) StopPodSandbox(ctx context.Context, r *runtime.St
 	id := sandbox.ID
 
 	// Teardown network for sandbox.
-	_, err = c.os.Stat(sandbox.NetNS)
+	netNS := sandbox.NetNsPath()
+	_, err = c.os.Stat(netNS)
 	if err == nil {
-		if teardownErr := c.netPlugin.TearDownPod(sandbox.NetNS, sandbox.Config.GetMetadata().GetNamespace(),
+		if teardownErr := c.netPlugin.TearDownPod(netNS, sandbox.Config.GetMetadata().GetNamespace(),
 			sandbox.Config.GetMetadata().GetName(), id); teardownErr != nil {
 			return nil, fmt.Errorf("failed to destroy network for sandbox %q: %v", id, teardownErr)
 		}
 	} else if !os.IsNotExist(err) { // It's ok for sandbox.NetNS to *not* exist
 		return nil, fmt.Errorf("failed to stat netns path for sandbox %q before tearing down the network: %v", id, err)
 	}
+	if err = sandbox.NetNSRemove(); err != nil {
```

Comment:
Add TODO that, it's still possible that `cri-containerd` crashes after we teardown the network, but before we remove the network namespace.

In that case, we'll not be able to remove the sandbox anymore. The chance is slim, but we should be aware of that.

In the future, once `TearDownPod` is idempotent, this will be fixed.